### PR TITLE
(new package) vlc [AU vlc]

### PR DESCRIPTION
**Pull request by majkinetor**
Moved from @ferventcoder : https://github.com/ferventcoder/chocolatey-packages/pull/205

### CODE REVIEW

**Comment by majkinetor** [comment]
I will merge when @ferventcoder confirms it.

File: automatic/vlc/update.ps1 [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import-module au
+
+$releases = 'http://www.videolan.org/vlc/download-windows.html'
```
Comment (AdmiringWorm) [inline automatic/vlc/update.ps1]: why not use https?`

File: automatic/vlc/vlc.nuspec [hunk 2]
```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata>
+    <id>VLC</id>
+    <title>VLC media player</title>
+    <version>2.2.4</version>
+    <authors>VideoLAN Organization</authors>
+    <owners>Rob Reynolds</owners>
```
Comment (AdmiringWorm) [inline automatic/vlc/vlc.nuspec]: should have chocolatey in the owners element

Comment (gep13) [inline automatic/vlc/vlc.nuspec]: Agreed.  If once you are both happy with this, if you assign to me, I can get the chocolatey user added to that package.

Comment (majkinetor) [inline automatic/vlc/vlc.nuspec]: You should do that now for this package and php

**Comment by gep13** [comment]
The chocolatey user has been added on chocolatey.org for this package.

**Comment by majkinetor** [comment]
@gep13 

when you see `[AU ...]` etc. in the description this means that you should specify that in commit message when merging.

**Comment by majkinetor** [comment]
https://github.com/chocolatey/chocolatey-coreteampackages/commit/ebdf71941c87aa71a54ca382e2e1d5fe734903ec

**Comment by gep13** [comment]
@majkinetor doh! My bad, missed that.

**Comment by majkinetor** [comment]
No problem, I wasn't sure if you knew for our deal :)

**Pull request merged**